project/.travis.yml.twig

```diff
@@ -50,15 +50,17 @@ matrix:
 
 before_install:
   - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" /tmp; fi;
-  - composer config --quiet --global github-oauth.github.com $GITHUB_OAUTH_TOKEN
-  - composer global require phpunit/phpunit:@stable --no-update
-  - sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
-  - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+  - if [ "$TARGET" = 'test' ]; then composer config --quiet --global github-oauth.github.com $GITHUB_OAUTH_TOKEN; fi;
+  - if [ "$TARGET" = 'test' ]; then sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json; fi;
+  - if [ "$TARGET" = 'test' -a "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
 
 install:
-  - travis_wait composer global update --prefer-dist --no-interaction
-  - travis_wait composer update --prefer-dist --no-interaction --prefer-stable $COMPOSER_FLAGS
-  - pip install -r {{ docs_path }}/requirements.txt --user `whoami`
+  - if [ "$TARGET" = 'test' ]; then mkdir "$HOME/bin"; fi;
+  - if [ "$TARGET" = 'test' ]; then if [[ "$TRAVIS_PHP_VERSION" < '5.6' ]]; then wget https://phar.phpunit.de/phpunit-old.phar --output-document="$HOME/bin/phpunit"; fi; fi;
+  - if [ "$TARGET" = 'test' ]; then if [[ ! "$TRAVIS_PHP_VERSION" < '5.6' ]]; then wget https://phar.phpunit.de/phpunit.phar --output-document="$HOME/bin/phpunit"; fi; fi;
```

Comment:
``[[ "$TARGET" = 'test' &&  "$TRAVIS_PHP_VERSION" < '5.6' ]] && wget https://phar.phpunit.de/phpunit-old.phar --output-document="$HOME/bin/phpunit" `` should work here